### DISCUSSION TITLE
Easiest way to bring a value from Loxone into HA

### DISCUSSION THREAD

**Discussion by ThePhantom79** [JoDehli/PyLoxone#337] [Q&A]
I have several (for pyloxone) not supported entities in loxone. 
Example: Sauna block. I now only want to transfer the temperature into HA - but I dont know how to do it. Should I put the value into a "merker" (I dont know english word for this), or some other thing? Or into an "InfoOnlyAnalog" (that is supported for example - but what is this? Or "text input"? ) Can someone give me a hint?

**Comment by JoDehli** [comment]
@ThePhantom79 you can use a InfoOnlyAnalog. You must set it visible that home-assistant discovers the InfoOnlyAnalog.
You can give also the right unit so that pyloxone recognize it as a temperature sensor.

**Comment by ThePhantom79** [comment]
Thanks for the answer ... but the question is: What is "InfoOnlyAnalog"? There is no "building block" or something esle, what is calles "InfoOnlyAnalog" in Loxone ...

**Reply by JoDehli** [reply]
@ThePhantom79 I think it is a Merker with a float or int value. You must try it.

**Reply by ThePhantom79** [reply]
I tried a lot - but with no success. I think "InfoOnlyAnalog" is a characteristic of a virtual input (you can tick "status only" - then it is an InfoOnly Object.) But this doesnt help me - as I need an output object to which I can send my value and that can be read by pyloxone)

**Reply by ThePhantom79** [reply]
I  tried and read a lot - imho it is not possible.
My temperature value comes from an RS485 actor. This actor do not have an UUID - so even with an custom sensor that I tried to create by my self failed, as I do not have this UUID. And that is mandatory. 
If you dont have a virtual input for your sensor, you will not get it into pyloxone :-(
Also a "Merker" is not shown as entity in pyhole unfortunataly (you cant find any merker in the loxone json config file)

![image](https://github.com/user-attachments/assets/e04b6d0a-62c9-4663-b11c-11e98338e879)

**Reply by ThePhantom79** [reply]
After trying a lot ... it works now.
You can use "Merker" to be read by PyLoxone. You have to make them visible (as you said), but also they have to have a room and a category configured in LoxConfig. If you use "do not use" in the drop down box, it will not be visible by pyloxone - even if the "visible in visualisation" box is checked ... Maybe this helpe someone else. With this I can now transfer every valie to HomeAssistant ...

**Reply by JoDehli** [reply]
@ThePhantom79 ok I did not now that you have not room or category assigned. This really needed to be shown up in Homeassistent. Me be you can add it to the readme section as little hint?

**Reply by gusa1234** [reply]
![Uploading Bildschirmfoto 2026-08-21 um 07.18.42.png…]()
Hi @JoDehli and @ThePhantom79, sorry for bringing up this older discussion.

I seem to have exactly the same problem. My analog marker is enabled for visualization, has both a room and a category assigned, and the configuration has been saved to the Miniserver. The Home Assistant user is an administrator with full permissions.

Nevertheless, the marker is completely absent from `/data/LoxAPP3.json`, while another apparently identical marker is included as `InfoOnlyAnalog`.

Do you remember whether there was any additional step that finally made it work? Did you copy an existing working marker, recreate it, use a Virtual Status, or change another setting not mentioned in your comment?

Any detail you still remember would be very helpful. Thank you!

**Comment by JoDehli** [comment]
@ThePhantom79 I think this discussion can be closed?
